tests/ui/perfherder/alerts-view/alerts_test.jsx

```diff
@@ -205,21 +202,15 @@ test('toggle buttons should filter alert summary and alerts by selected filter',
   expect(alert1).toBeInTheDocument();
   expect(alert2).toBeInTheDocument();
 
-  expect(hideImprovements).not.toHaveClass('active');
   expect(hideDownstream).not.toHaveClass('active');
 
-  // one filter selected
-  fireEvent.click(hideImprovements);
-
-  expect(hideImprovements).toHaveClass('active');
-  expect(alertSummary1).not.toBeInTheDocument();
   expect(alertSummary2).toBeInTheDocument();
   expect(alert1).toBeInTheDocument();
   expect(alert2).toBeInTheDocument();
 
   // second filter selected
   fireEvent.click(hideDownstream);
-  expect(alertSummary1).not.toBeInTheDocument();
```

Comment:
We should change this from `.not.toBeInTheDocument()` to  `.toBeInTheDocument()` since the previous filter is not applied anymore.